src/validate_pyproject/plugins/distutils.schema.json

```diff
@@ -1,14 +1,15 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
 
-  "$id": "https://docs.python.org/3/install/",
+  "$id": "https://setuptools.pypa.io/en/latest/deprecated/distutils/configfile.html",
   "title": "``tool.distutils`` table",
   "$$description": [
```

Comment:
Why is this `$$description`? As far as I can tell, such as with https://json-schema.org/understanding-json-schema/reference/annotations, this field is supposed to be `description`, and schema store's validator doesn't like `$$description`.

Reply:
Ahh, this is a sphinx-jsonschema addition just to allow a list of strings instead of a string (at the expense of it not being standard)?

Reply:
Yes that is the reason.

It is very difficult to write long strings in JSON.

We could generate the schemas by using YAML, or other solutions, but the problem is to take in more dependencies.

Since I designed the project to be compatible with build backends, I try to keep the dependencies at bay...

Reply:
There's always pre-processing, where you write a something.yml and convert it to the json schema, but do that in the repo. I did this with cibuildwheel, https://github.com/pypa/cibuildwheel/blob/main/bin/generate_schema.py (and nox job). That avoids a runtime dependency, only a development dependency.

But that's fine, I have to process this anyway.